config/default.js

```diff
@@ -47,5 +48,11 @@ module.exports = {
     enabled: true,
     defaultUser: null,
     secret: 'top secret 51'      // DO NOT USE THIS IN PRODUCTION
+  },
+  ssltunnel: {
+    enabled: true,
+    registration_endpoint: 'mozilla-iot.org',
+    domain: 'box.mozilla-iot.org',
```

Comment:
Why is this box.mozilla-iot.org ? It should be mozilla-iot.org. Was there a reason you weren't able to remove the box prefix?